Source/WebCore/rendering/RenderBox.cpp

```diff
@@ -1534,6 +1535,16 @@ bool RenderBox::hitTestClipPath(const HitTestLocation& hitTestLocation, const La
 
     auto offsetFromHitTestRoot = toLayoutSize(accumulatedOffset + location());
     auto hitTestLocationInLocalCoordinates = hitTestLocation.point() - offsetFromHitTestRoot;
+
+    auto hitsClipContent = [&](Element& element) -> bool {
+#if ENABLE(LAYER_BASED_SVG_ENGINE)
+        if (auto* clipper = dynamicDowncast<RenderSVGResourceClipper>(element.renderer()))
```

Comment:
The negation is either wrong or confusing - I guess the former.

Reply:
I think it is wrong too, let's see the bots results.